.bazelrc

```diff
@@ -1,2 +1,3 @@
 build --verbose_failures
 build --sandbox_debug
+build --incompatible_depset_is_not_iterable=false
```

Comment:
In case this is caused by `rules_proto`, you can upgrade to latest master where this is fixed.

Reply:
@Yannic I needed this for something out of protobuf if memory serves. I think this is upgraded to `rules_proto` latest but I'll make sure